caravel/storage/entities.py

```diff
@@ -55,6 +58,7 @@ def keywords(self):
         # Tokenize title and body (ranking them equally)
         words = self.title.split() + self.body.split()
```

Comment:
I don't think can access self.seller here. Were you saying to deal with this in the caller function so only pass in non-email addresses to fold_query_term()?